resources/views/components/link.blade.php

```diff
@@ -0,0 +1,7 @@
+<a {{
+        $attributes->merge(['target' => 'blank', 'rel' => 'noopener'])
+        // these styles are not set globally because they'll break most existing links, which only have the default color set
+        ->class('text-pine-600 hover:text-pine-500 active:text-pine-700')
+    }}>
+    {{ $slot }}
+</a>
```

Comment:
Blade unfortunately has no way to ignore spaces around tags, so we should do this to avoid unwanted spaces after links:

![image](https://github.com/user-attachments/assets/b63e95b1-3570-4396-86f4-792437b92820)

```suggestion
<a {{
    $attributes->merge(['target' => 'blank', 'rel' => 'noopener'])
    // these styles are not set globally because they'll break most existing links, which only have the default color set
    ->class('text-pine-600 hover:text-pine-500 active:text-pine-700 inline')
}}>{{ $slot }}</a>
```

Reply:
Btw that file didn't end with a line break (I think), but it should according to .editorconfig. So I would add this rule, ideally, for the same reason as above:

```
[*.blade.php]
insert_final_newline = false
```